src/main/java/uk/gov/companieshouse/web/pps/controller/pps/ConfirmationController.java

```diff
@@ -69,7 +69,7 @@ public String getConfirmation(@PathVariable String companyNumber,
             LOGGER.info("Payment status is " + paymentStatus + " and not of status 'paid', returning to beginning of journey");
             PayableLateFilingPenalty payableLateFilingPenalty;
             try {
-                payableLateFilingPenalty = payableLateFilingPenaltyService
+                payableLateFilingPenalty = payablePenaltyPaymentService
```

Comment:
What about even just reverting this for now and have it addressed in a separate ticket? The codebase will have to be made generic, so it might be worth covering that in a ticket, and we can split out the LFP journey and then make rename these.

Reply:
Currently changed to PayablePenaltyService, happy to discuss about that.

Reply:
Think getting that as is now is fine, we can always rename it later but yeah, it's "generic" now at least.